021-Features/075-File-attachments.mdx

```diff
@@ -89,18 +89,18 @@ You can use the public URL directly without writing code. For instance:
 https://us-east-1.storage.xata.sh/4u1fh2o6p10blbutjnphcste94 is available publicly
 
 <TabbedCode tabs={['TypeScript', 'Python']}>
-
-```ts
-// Allow this user's photo to be accessed publicly at any time by the URL alone
-const user = await xata.db.Users.update('record_id', {
-  photo: {
-    enablePublicUrl: true
-  }
-});
+  ```ts
+  // Allow this user's photo to be accessed publicly at any time by the URL alone
+  const user = await xata.db.Users.update('record_id', {
+    photo: {
+      enablePublicUrl: true
+    }
+  });
 
 // Retrieve the public URL on the file in the "photo" column
 const { url } = user.photo.transform({ quality: 50 });
-```
+
+````
```

Comment:
Double check your editor. These quadruple ```` keep getting added and will break the layout. Easiest way to avoid this (which are likely happening in VSCode due to a linting setting) is to keep a line break between `<TabbedCode>` and your fences. Also, to avoid unnecessary indentation in the code blocks, try to avoid starting code blocks with indentation.